bingo/postgres/src/pg_common/bingo_pg_common.cpp

```diff
@@ -19,18 +22,14 @@ extern "C"
 
 #include "base_c/bitarray.h"
 #include "base_cpp/output.h"
-#include "base_cpp/scanner.h"
 #include "base_cpp/tlscont.h"
+
 #include "bingo_core_c.h"
+
 #include "bingo_pg_buffer.h"
 #include "bingo_pg_common.h"
 #include "bingo_pg_config.h"
 #include "bingo_pg_ext_bitset.h"
-#include "bingo_pg_text.h"
-#include "molecule/molecule.h"
-#include "molecule/molecule_auto_loader.h"
-#include "molecule/molecule_fingerprint.h"
-#include <math.h>
```

Comment:
Here also many include files are removed. Is it subject of this pull request?

Reply:
after reordering some includes became redundant and so've been deleted.